pages/developers/gha_basic.md

```diff
@@ -56,6 +56,8 @@ lint:
   steps:
     - uses: actions/checkout@v3
     - uses: actions/setup-python@v4
+      with:
+        python-version: "3.x"
```

Comment:
```suggestion
        python-version: ">=3.7"
```
Something like this would be fine too, but would require yearly bumping of the value.